src/import/FSHTank.ts

```diff
@@ -107,4 +71,96 @@ export class FSHTank {
+    // Resolve alias if necessary
+    item = this.resolveAlias(item) ?? item;
+
+    // No types passed in means to search ALL supported types
+    if (types.length === 0) {
+      types = [Type.Profile, Type.Extension, Type.ValueSet, Type.CodeSystem, Type.Instance];
+    }
+
+    for (const t of types) {
+      let result;
+      switch (t) {
+        case Type.Profile:
+          result = this.getAllProfiles().find(
+            p =>
+              p.name === item ||
+              p.id === item ||
+              `${this.config.canonical}/StructureDefinition/${p.id}` === item
+          );
+          break;
+        case Type.Extension:
+          result = this.getAllExtensions().find(
+            e =>
+              e.name === item ||
+              e.id === item ||
+              `${this.config.canonical}/StructureDefinition/${e.id}` === item
+          );
+          break;
+        case Type.ValueSet:
+          result = this.getAllValueSets().find(
+            vs =>
+              vs.name === item ||
+              vs.id === item ||
+              `${this.config.canonical}/ValueSet/${vs.id}` === item
+          );
+          break;
+        case Type.CodeSystem:
+          result = this.getAllCodeSystems().find(
+            vs =>
+              vs.name === item ||
+              vs.id === item ||
+              `${this.config.canonical}/CodeSystem/${vs.id}` === item
+          );
+          break;
+        case Type.Instance:
+          result = this.getAllInstances().find(i => i.name === item || i.id === item);
+          break;
+        case Type.Resource:
+        case Type.Type:
+        default:
+          // Tank doesn't support these types
+          break;
+      }
+      if (result != null) {
+        return result;
+      }
+    }
+    // No match, return undefined
+    return;
+  }
+
+  fishForMetadata(item: string, ...types: Type[]): Metadata | undefined {
+    const result = this.fish(item, ...types);
+    if (result) {
+      const meta: Metadata = {
+        id: result.id,
+        name: result.name
+      };
+      if (result instanceof Profile || result instanceof Extension) {
+        meta.url = `${this.config.canonical}/StructureDefinition/${result.id}`;
+        meta.parent = result.parent;
+      } else if (result instanceof FshValueSet) {
+        meta.url = `${this.config.canonical}/ValueSet/${result.id}`;
+      } else if (result instanceof FshCodeSystem) {
+        meta.url = `${this.config.canonical}/CodeSystem/${result.id}`;
+      } else if (result instanceof Instance) {
+        meta.instanceOf = result.instanceOf;
+      }
+      return meta;
+    }
+    return;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  fishForFHIR(item: string, ...types: Type[]): any | undefined {
+    // the FSHTank cannot return FHIR definitions, but we define this function
+    // in order to implement the Fishable interface
+  }
```

Comment:
I think I prefer to go ahead and leave the parameters in.  I'm surprised that typescript still considers it as implementing `Fishable` when the function signature changed.  But let's just keep it as-is in case that's a bug in TypeScript. ;-)